CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.1.22
+
+- Update default value functionality to select list
```

Comment:
@lavanya-bmw please be more specific
I suggest we always name first of all the component - in this case "SelectList component..." and afterwards the change. Is it really an update or did you actually added the functionality that the select list has an defect value set?

Last but not least: has this change an impact of current used selectList components?

Reply:
Since default value was not working for selectlist, this fix was made